collectors/scm/github/src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java

```diff
@@ -166,9 +166,20 @@ public void collect(Collector collector) {
         int commitCount = 0;
         int pullCount = 0;
         int issueCount = 0;
-
+       
         clean(collector);
-        for (GitHubRepo repo : enabledRepos(collector)) {
+        
+        String proxyUrl = gitHubSettings.getProxy();
+        String proxyPort = gitHubSettings.getProxyPort();
+ 
+        if (!StringUtils.isEmpty(proxyUrl) && !StringUtils.isEmpty(proxyPort)) {
+            System.setProperty("http.proxyHost", proxyUrl);
```

Comment:
Nice work here. One more thing can you add two more property options? `proxyUser` and `proxyPassword ` here and place it in a StringUtils.isEmpty check? Similar to Line 175. Some users may need to provide authentication.